gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -72,6 +77,42 @@ local function VoiceTryDisable()
     return false
 end
 
+local function VoiceToggle()
+    if VOICE.IsSpeaking() then
+        return VoiceTryDisable()
+    else
+        return VoiceTryEnable()
+    end
+end
+
+VOICE.ActivationModes = {
+    ptt = { OnPressed = VoiceTryEnable, OnReleased = VoiceTryDisable, OnJoin = VoiceTryDisable },
+    ptm = { OnPressed = VoiceTryDisable, OnReleased = VoiceTryEnable, OnJoin = VoiceTryEnable },
+    toggle_disabled = { OnPressed = VoiceToggle, OnJoin = VoiceTryDisable },
+    toggle_enabled = { OnPressed = VoiceToggle, OnJoin = VoiceTryEnable },
+}
+
+VOICE.ActivationModeChoices = util.ComboBoxChoicesFromKeys(
+    VOICE.ActivationModes,
+    "label_voice_activation_mode_",
+    cvVoiceActivationMode:GetString()
+)
+
+---
+-- @param string func The name of the function to call on the current voice activation mode
+-- @return function A closure that calls the function on the current voice activation mode if it exists
+-- @realm client
+function VOICE.ActivationModeFunc(func)
```

Comment:
maybe rename it to `functionName` to make it more clear what its purpose is. `func` sounds like a function reference which this isn't. It probably is even more clear if you just call it `mode` as this is basically what this does.